### PR TITLE
UI Setting Screen

### PR DESCRIPTION
**Pull request by AbdullahAhmedSoomro**
- Setting Screen Completed.

I use the Setting tile that I created in this issue #314 , this PR is not merged yet so I simply copy this Setting tile and Paste in the Setting screen to make some progress in this issue, so when issue  #314 is merged, I will remove the Setting tile that i copy paste in Setting Screen and use this one issue #314

### CODE REVIEW

**Comment by AbdullahAhmedSoomro** [comment]
Pending:
- Images

File: lib/presentation/screen/setting_screen.dart [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+            style: Theme.of(context)
+                .textTheme
+                .bodyMedium!
+                .copyWith(fontWeight: FontWeight.w400),
+            children: [
+              TextSpan(
+                text: 'Blocship',
+                style: Theme.of(context).textTheme.bodyMedium!.copyWith(
+                      fontWeight: FontWeight.w400,
+                      color: const Color(0xFF1F1F1F),
```
Comment (hashirshoaeb) [inline lib/presentation/screen/setting_screen.dart:88]: why color is not defined in theme?

**Review by hashirshoaeb — APPROVED** [review]

File: lib/presentation/screen/setting_screen.dart (hunk 1 above)
Comment (AbdullahAhmedSoomro) [reply lib/presentation/screen/setting_screen.dart:88]: Color will be implemented from the theme after completing this issue #298

Comment (hashirshoaeb) [reply lib/presentation/screen/setting_screen.dart:88]: ok

Comment (hashirshoaeb) [reply lib/presentation/screen/setting_screen.dart:88]: move to merging then

Comment (AbdullahAhmedSoomro) [reply lib/presentation/screen/setting_screen.dart:88]: Okay

Comment (AbdullahAhmedSoomro) [reply lib/presentation/screen/setting_screen.dart:88]: Mene Simple Scaffold Lagaya howa tha ab change krdya hai Gradient Scaffold laga dia hai Theek hai?

**Comment by AbdullahAhmedSoomro** [comment]
Screenshot_20230213-162246

**Comment by AbdullahAhmedSoomro** [comment]
- Padding Added from padding Class

Screenshot_20230213-164008

**Pull request merged**